packages/database/lib/migrations/20250723214519_create_pre_provisioned_providers.cjs

```diff
@@ -0,0 +1,20 @@
+/**
+ * @param {import('knex').Knex} knex
+ */
+exports.up = async function (knex) {
+    return knex.schema.createTable('_nango_pre_provisioned_providers', function (table) {
```

Comment:
we don't use _nango prefix anymore, I would suggest `providers_shared_credentials`